airflow/bin/cli.py

```diff
@@ -185,6 +185,56 @@ def trigger_dag(args):
     logging.info(message)
 
 
+def batch_set_pools(args):
+    """
+    set a batch of Airflow pools
```

Comment:
it would be good to put more details in docstring(e.g " allow us to set a batch of airflow pools within one session commits.")